radar_compose.yaml

```diff
@@ -0,0 +1,36 @@
+services:
+
+  ros1_master:
+    image: "ghcr.io/ipab-rad/av_radar:1.0.0"
+    container_name: ros1_master
+    network_mode: "host"
+    privileged: true
+    command: rosmaster --core
+    volumes:
+      - "/dev/shm:/dev/shm"
+
+  av_radar:
+    image: "ghcr.io/ipab-rad/av_radar:1.0.0"
+    container_name: av_radar
+    network_mode: "host"
+    privileged: true
+    depends_on:
+      - ros1_master
+
+  ros1_bridge:
+    image: "ghcr.io/ipab-rad/ros1_bridge:1.0.0"
+    container_name: ros1_bridge
+    network_mode: "host"
+    privileged: true
+    depends_on:
+      - av_radar
```

Comment:
Major nitpick: This container **will** need to share the shm volume since it is a ROS1-ROS2 bridge and does run ROS2.